### PR TITLE
[CRN 1043] alumni badge in shared research list/details

### PR DESCRIPTION
**Pull request by ricardocarvalh0**

### CODE REVIEW

File: packages/react-components/src/molecules/UsersList.tsx [hunk 1]
```diff
@@ -73,6 +82,9 @@ const UsersList: FC<UsersListProps> = ({
                 <Avatar {...user} imageUrl={user.avatarUrl} />
               </ImageLink>
               <span css={nameStyles}>{user.displayName}</span>
+              {user.alumniSinceDate && (
+                <span css={iconStyles}>{alumniBadge}</span>
```
Comment (oratiu) [inline packages/react-components/src/molecules/UsersList.tsx:86]: Do we need the span here ?

File: packages/react-components/src/molecules/__tests__/UsersList.test.tsx [hunk 2]
```diff
@@ -54,6 +54,48 @@ it('does not link external users', () => {
...
 });
 
+describe('alumni badge', () => {
+  it('shows alumni badge for internal and alumni users', () => {
+    const { getByText } = render(
+      <UsersList
+        max={1}
+        users={Array.from({ length: 1 }).map(() => ({
+          ...createUserResponse(),
+          displayName: 'John Doe',
```
Comment (kmaid) [inline packages/react-components/src/molecules/__tests__/UsersList.test.tsx:64]:
```suggestion

```

File: packages/react-components/src/molecules/__tests__/UsersList.test.tsx [hunk 3]
```diff
@@ -54,6 +54,48 @@ it('does not link external users', () => {
...
+          alumniSinceDate: new Date(2021, 6, 12, 14, 32).toISOString(),
+        }))}
```
Comment (kmaid) [inline packages/react-components/src/molecules/__tests__/UsersList.test.tsx:66]:
```suggestion
        users={[{
          ...createUserResponse(),
          alumniSinceDate: new Date(2021, 6, 12, 14, 32).toISOString(),
        }]}
```

File: packages/react-components/src/molecules/__tests__/UsersList.test.tsx [hunk 4]
```diff
@@ -54,6 +54,48 @@ it('does not link external users', () => {
...
+    );
+    expect(getByText('Alumni Badge')).toBeInTheDocument();
+  });
+  it('does not show alumni badge if user is not alumni', () => {
+    const { queryByText } = render(
+      <UsersList
+        max={1}
+        users={Array.from({ length: 1 }).map(() => ({
+          ...createUserResponse(),
+          displayName: 'John Do',
```
Comment (kmaid) [inline packages/react-components/src/molecules/__tests__/UsersList.test.tsx:77]: Same as above

File: packages/react-components/src/molecules/__tests__/UsersList.test.tsx [hunk 5]
```diff
@@ -54,6 +54,48 @@ it('does not link external users', () => {
...
+  it('does not show alumni badge for external users', () => {
+    const { queryByText } = render(
+      <UsersList
+        max={1}
+        users={Array.from({ length: 1 }).map(
+          () =>
+            ({
+              displayName: 'John Do',
+            } as ExternalAuthorResponse),
+        )}
```
Comment (kmaid) [inline packages/react-components/src/molecules/__tests__/UsersList.test.tsx:92]: Cant it just be { displayName: blah, id: blah}

File: packages/react-components/src/molecules/__tests__/UsersList.test.tsx [hunk 6]
```diff
@@ -74,22 +72,19 @@ describe('alumni badge', () => {
...
     const { queryByText } = render(
       <UsersList
         max={1}
-        users={Array.from({ length: 1 }).map(
-          () =>
-            ({
-              displayName: 'John Do',
-            } as ExternalAuthorResponse),
-        )}
+        users={Array.from({ length: 1 }).map(() => ({
```
Comment (kmaid) [inline packages/react-components/src/molecules/__tests__/UsersList.test.tsx:84]: You don't need to use array from length 1. You can just provide [{...user}]

**Review by kmaid — APPROVED** [review]

**Pull request merged**